src/content/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -420,7 +467,7 @@ You can create, read, update, or delete obfuscation rules by using the New Relic
       <img
         title="Create an obfuscation rule using the Logs UI"
         alt="Create an obfuscation rule using the Logs UI"
-        src={obfuscationCreateRuleUi}
+        src="/Users/lchapman/desktop/ObfuscationCreateExpression.png"
```

Comment:
@lchapman4 it looks like this is a local image that needs to be uploaded to `src/images`.

Reply:
the `src` value here should reference the variable for the relevant `import` at the top of the file. 

the image that was also added should be imported with the filename matching the filename of the image

Reply:
@lchapman4 I just pushed up a change that should fix this.